src/main/webapp/js/issues-detail.js

```diff
@@ -165,12 +164,14 @@
     var activeTab = localStorage.getItem('activeTab');
     if (activeTab) {
         detailsTabs.find('a[href="' + activeTab + '"]').tab('show');
+    } else {
```

Comment:
This will not work if the tab is not shown (i.e. active is set but not visible). I fixed it differently in https://github.com/jenkinsci/analysis-core-plugin/commit/911797ef70ffbb6a49e3192a2e83e0175c5455a6